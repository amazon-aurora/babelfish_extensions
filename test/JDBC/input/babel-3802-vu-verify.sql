EXECUTE babel_3802_vu_prepare_t_p1
GO

EXECUTE babel_3802_vu_prepare_t_p2
GO

EXECUTE babel_3802_vu_prepare_t_p3
GO

SELECT * FROM babel_3802_vu_prepare_t_v1
GO

SELECT * FROM babel_3802_vu_prepare_t_v1
GO