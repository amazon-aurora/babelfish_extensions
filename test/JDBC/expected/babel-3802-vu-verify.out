EXECUTE babel_3802_vu_prepare_t_p1
GO
~~START~~
int
1000000
~~END~~


EXECUTE babel_3802_vu_prepare_t_p2
GO
~~START~~
int#!#int#!#int
-32768#!#32767#!#0
~~END~~


EXECUTE babel_3802_vu_prepare_t_p3
GO
~~START~~
int#!#int#!#int
0#!#255#!#0
~~END~~


SELECT * FROM babel_3802_vu_prepare_t_v1
GO
~~START~~
int#!#int#!#int
-32768#!#32767#!#0
~~END~~


SELECT * FROM babel_3802_vu_prepare_t_v1
GO
~~START~~
int#!#int#!#int
-32768#!#32767#!#0
~~END~~

